### ISSUE TITLE
[Bug Report] Can't open or use Amulet v10.x

### ISSUE THREAD

**Issue by Delvin4519** [Amulet-Team/Amulet-Map-Editor#932]
### Current Behaviour:

Open world menu results in an error and cannot open world.

### Expected behavior:

Open world menu should allow selection of world.

### Steps To Reproduce:

Open world menu in Amulet v10.x. Essentially right after startup when selection at the menu screen.

### Environment:

- Amulet Version: v10.x all versions since v10.0 through v10.13 
- 
Windows 10 v21H2, Java/Bedrock any minecraft world version, I cannot select a world at any menu since it errors before selection.

### Screenshots

![image](https://github.com/Amulet-Team/Amulet-Map-Editor/assets/23464599/df54bd62-91b8-4b64-b5d6-37e2b2b61269)

### Worlds

I can't select any world at all since the program errors at the open world menu.

**Comment by gentlegiantJGC** [comment]
Please can you share a world that this happens in

**Comment by Delvin4519** [comment]
This affects all worlds, I can't access any world since the program results in error when the button to open world is selected.

**Comment by gentlegiantJGC** [comment]
I haven't had this issue with any world I have created. Please can you share a world this happens in so I can debug the issue.
A newly created world that has the issue is good.

**Comment by Delvin4519** [comment]
I figured out the issue.

Looking at the error message, Amulet v10.15 literally locks out from loading at all when a level.dat file is missing the "LastPlayed" tag, preventing any world from loading if the said world is at least in the main Java directory (idk about bedrock). This results in unable to use Amulet or identify said world directly since the select world screen fails to load. The error message won't specify the problematic world that causes the issue.

I have a world intentionally empty with a blank and empty level.dat, in the original Java directory (Java MC v1.12.2), so I could load Java (1.8.0 to) 1.12.2 customized worlds in the AMIDST biome map viewer program.

This empty world results in Amulet v10.15 (down to v10.0) locking out, when placed in the main Java world directory.

[zz copy.zip](https://github.com/Amulet-Team/Amulet-Map-Editor/files/11937278/zz.copy.zip)

If I add a LastPlayed tag, then it seems to have fixed the issue.

[zz copy.zip](https://github.com/Amulet-Team/Amulet-Map-Editor/files/11937264/zz.copy.zip)

The bug in Amulet existed previously and I had the issue previously, before the issue got fixed in the 0.9.0x Amulet releases, and then the bug came back again in newer versions.
